### ISSUE TITLE
Barometer dial not showing pressure.

### ISSUE THREAD

**Issue by trudK45** [tom-r/tactics_pi#42]
I have tested the Barometer instrument an it seem to work with the digital display and the histogram but not with the analog dial meter. I am sending/receiving string:
$IIXDR,P,1.01750,B,Barometer*25<0x0D><0x0A>

![Barometer](https://user-images.githubusercontent.com/10415016/65177553-2e946d80-da57-11e9-8271-72cc77f8141b.jpg)

**Comment by trudK45** [comment]
Follow up: Dial works in dashboard.

**Comment by ghost** [comment]
Hi, it is because of this setting in https://github.com/tom-r/tactics_pi/blob/master/src/dial.cpp starting from line 97:
```
  // Test value must be greater than 360 to enable some compass-type displays.
  if ((st == m_MainValueCap) && (data < 400.0))
...
  else if ((st == m_ExtraValueCap) && (data < 400.0))
...
```
set it back to 1200.0 which is the limit in Dashboard plug-in.

**Comment by rgleason** [comment]
Changed line 104  and added a note line.

  // Set line 104 back to 1200 to display barometer too
  if ((st == m_MainValueCap) && (data < 400.0))
  {
    m_MainValue = data;
    m_MainValueUnit = unit;
  }
  else if ((st == m_ExtraValueCap) && (data < 1200.0))
  {
    m_ExtraValue = data;
    m_ExtraValueUnit = unit;
  }
}

Done, should work.

**Comment by rgleason** [comment]
Fixed in v1.0.28  please close @trudK45

**Comment by trudK45** [comment]
Will check this when I get back to the boat in 3-4 weeks.

**Comment by rgleason** [comment]
Thanks. It has been corrected in version 1.0.28 which will be in PIM shortly.
